plugins/git/committia.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  config,
+  helpers,
+  pkgs,
+  ...
+}:
+with lib;
+with helpers.vim-plugin;
+  helpers.vim-plugin.mkVimPlugin config {
+    name = "committia-vim";
+    description = ''
+      When you type git commit, Vim starts and opens a commit buffer. This plugin improves the commit buffer.
+
+      committia.vim splits the buffer into 3 windows; edit window, status window and diff window.
+      You no longer need to repeat moving to another window, scrolling and backing to the former position in order to see a long commit diff.
+    '';
+    package = pkgs.vimPlugins.committia-vim;
+    extraPackages = [pkgs.git];
```

```suggestion
    package = pkgs.vimPlugins.committia-vim;
   globalPrefix = "committia";
    extraPackages = [pkgs.git];
```

Can you also rebase on `main` please ?

---

Added the global prefix and rebased as requested.